plugins/a11y-text-wand/index.js

```diff
@@ -22,11 +22,11 @@ class A11yTextWand extends Plugin {
         this.summary(" ");
         this.panel.render();
 
-        $(document).on("mousemove.wand", function(e) {
-            let element = document.elementFromPoint(e.clientX, e.clientY);
+        $(document).on("mousemove.wand", function(e, testElement) {
+            let element = (testElement) ? testElement : document.elementFromPoint(e.clientX, e.clientY);
 
             let textAlternative = axs.properties.findTextAlternatives(
-                element, {});
+                element, {}, true);
```

Comment:
This was the real fix to the issue, the `true` tells findTextAlternatives to run recursively on the Element and it's children. Without this, nested `aria-hidden` elements will show up in the Screen Reader text that is displayed in tota11y's info panel.